piston/resource.py

```diff
@@ -244,15 +244,14 @@ def _regex_domain_match(self, origin):
                 return origin
 
     def options(self):
-        obj = self._get_obj_or_none()
-        allowed_methods = ', '.join((method.upper() for method in self.get_allowed_methods(obj)))
-
-        http_headers = {'Allowed': allowed_methods}
-        if getattr(settings, 'PISTON_CORS', False):
-            http_headers.update({
-                ACCESS_CONTROL_ALLOW_METHODS: allowed_methods,
-                ACCESS_CONTROL_ALLOW_HEADERS: ', '.join(self._get_cors_allowed_headers()),
-            })
+        if getattr(settings, 'PISTON_CORS', False) and self.request.META.get('HTTP_ORIGIN'):
+            http_headers = {
+                ACCESS_CONTROL_ALLOW_METHODS: self.request.META.get('HTTP_ACCESS_CONTROL_REQUEST_METHOD'),
```

Comment:
teoreticky by to tu mělo vrátit jen options, to je přesně to co se mi na cors nelíbilo